### ISSUE TITLE
Set `sidbebar_menu_foldabe = true` for better glancing experience

### ISSUE THREAD

**Issue by FOWind** [kubernetes/website#31817]
**This is a Feature Request**

**What would you like to be added**

Set `sidbebar_menu_foldabe = true` for Docsy `[params.ui]`
**Why is this needed**
Setting the `sidebar_menu_foldable = ture` can make a better experience for glancing the menu tree.

After setting `sidebar_menu_foldable = true` appearance shows like belowed:

![image](https://user-images.githubusercontent.com/13653453/154934459-fbef651c-99d1-484a-82ab-5dfc648a0c3f.png)

When I'm reading chapter `Kubernetes API` I can also expand the menu tree for looking for next chapter I want to read.

Current appearance shows like belowed:

![image](https://user-images.githubusercontent.com/13653453/154934521-73727d7e-acbe-480d-9ead-1974aebff615.png)

If I am reading chapter `Kubernetes API`, I can only see the menu under `Kubernetes API`. If I want to see other menu I must click the menu and enter a new page.

**Comment by sftim** [comment]
Hi @FOWind 

Why should we make this change? Can you explain the benefit for readers?

**Comment by FOWind** [comment]
1. For the new version, reader can only expand the menu tree for only glancing the indexes, and the old version can't do that.

   ![image](https://user-images.githubusercontent.com/13653453/155056343-11a33d3a-6dfc-4015-92a9-75af3c3b7d81.png)

2. After expanding the menu tree, readers can enter the deep level chapter they want to read directly without click the menu step by step.

   ![image](https://user-images.githubusercontent.com/13653453/155057222-c394d6a6-5a68-4564-bfd3-3d67136dc71d.png)

   For the old versions,  if you want to enter the deep level chapter, you should click the menu many times.

3. For the new version, the upper level menu is still dispalyed when enter deep level chapter, and the old version the upper level menu will disappear
   ![image](https://user-images.githubusercontent.com/13653453/155058141-316d72d1-d112-436a-9e83-74a1a5247e34.png)

  ![image](https://user-images.githubusercontent.com/13653453/155058304-cf6e7a1f-2c9a-402e-a346-976b2ba23436.png)

I think the new version is more convenient for me.

You can try it on https://deploy-preview-31818--kubernetes-io-main-staging.netlify.app/docs/home/

**Comment by sftim** [comment]
/triage accepted

I think this this is explained well enough to justify merging the change.
I wonder: would it be better to introduce the new UI with the v1.24 release, rather than arbitrarily?

**Comment by sftim** [comment]
@championshuttler there is already a PR open for this.

**Comment by tengqm** [comment]
#31818 is fixing this.

**Comment by sftim** [comment]
After #31818 merges, I'd like (nice to have) to see that change backported to older docs.

**Comment by FOWind** [comment]
https://github.com/kubernetes/website/pull/31818 has merged. 
Close.
